opencog/atoms/bind/PatternLink.cc

```diff
@@ -506,14 +548,8 @@ void PatternLink::unbundle_virtual(const std::set<Handle>& vars,
 		{
 			_pat.evaluatable_terms.insert(sh);
 			add_to_map(_pat.in_evaluatable, sh, sh);
-			// But they're virtual only if they have two or more
-			// unquoted, bound variables in them. Otherwise, they
-			// can be evaluated on the spot.
```

Comment:
I don't think you can evaluatable anything on the spot unless it's virtual to the top.  Otherwise it is supposed to be matched as it.

ie.  `1 < 2, 1 < x, x < y` could all potentially be matched as is and should not be evaluated on the spot without looking at what's above it.